src/python/pants/backend/awslambda/python/awslambda_python_rules.py

```diff
@@ -90,10 +96,17 @@ async def create_python_awslambda(
         pex_args=lambdex_args,
         input_files=merged_input_files,
         output_files=(pex_filename,),
-        description=f"Run Lambdex for {config.address.reference()}",
+        description=f"Processing {pex_filename}",
```

Comment:
Knowledge of lambdex is probably not something we need to impose on the user.

Reply:
I agree. Maybe mention AWS Lambda though? Something like "Creating AWS Lambda in {pex_filename}". "Processing app.zip" might not be obvious to the user.